requirements.txt

```diff
@@ -1,5 +1,7 @@
 biopython ; python_version == '2.7' or python_version >= '3.4'
 numpy
+scipy
+periodictable
```

Comment:
We have a periodic table thing in util: https://github.com/cclib/cclib/blob/master/cclib/parser/utils.py#L124

Do you think we should replace that with this package, or standardize on our own version?

Reply:
Our periodic table is not sufficient, because we need masses, specifically isotopic masses. I am not sure if `atommasses`, when present, are always isotopic masses. They should be when parsed from a frequency calculation. However I don't believe they're always available.

If there is a better Python package for this sort of data, and a better one for physical constants than scipy, I am open to suggestions.

We can eventually have our periodic table wrap an external package, once we pick an appropriate one, because holding our own atomic masses doesn't sound good.

Reply:
OK, I'm OK with this choice. `periodictable` look reasonable.